.config/brix/rust/template.rs.txt

```diff
@@ -0,0 +1,7 @@
+#[cfg(test)]
```

Comment:
not a big deal, but most templates have an extension for syntax highlighting. Might use that, I don't care, up to you.

Reply:
Yeah that's true I'll keep that in mind